distribution/docs/src/main/resources/_contents/_endpoints/catalog-rest-endpoint-contents.adoc

```diff
@@ -62,3 +77,86 @@ By default, the response body will include the XML representation of the Metacar
 |\${public_url}/services/catalog/sources/
 
 |===
+
+====== Sources Operation Example
+
+In the example below there is the local ${branding} distribution and a ${branding} OpenSearch federated source with id "${ddf-branding}-OS".
+
+.Sources Response Example
+[source,javascript,linenums]
+----
+[
+   {
+      "id" : "${ddf-branding}-OS",
+      "available" : true,
+      "contentTypes" :
+         [
+         ],
+      "version" : "2.0"
+   },
+   {
+      "id" : "ddf.distribution",
+      "available" : true,
+      "contentTypes" :
+         [
+         ],
+      "version" : "2.5.0-SNAPSHOT"
+   }
+]
+----
+
+Note that for all RESTful CRUD commands only one metacard ID is supported in the URL, i.e., bulk operations are not supported.
+
+===== Interacting with the REST CRUD Endpoint
+
+Any web browser can be used to perform a REST read.
```

Comment:
I'm finding conflicting sources for this one. Most of the advice to capitalize comes from older sources, though. It looks a little _self important_ to me, but I'm open to counter-arguments.